app/display/model/src/main/java/org/csstudio/display/builder/model/widgets/ChoiceButtonWidget.java

```diff
@@ -74,6 +77,15 @@ public ChoiceButtonWidget()
         super(WIDGET_DESCRIPTOR.getType(), 100, 43);
     }
 
+    private static ArrayWidgetProperty.Descriptor<WidgetProperty<String>> choiceItemDescriptor =
+            new ArrayWidgetProperty.Descriptor<>(WidgetPropertyCategory.BEHAVIOR, "items", "Items",
+                    (widget, index) -> {
```

Comment:
The 'index' is passed here so you can use that for the name instead of having to get the `size`.
So `(size+1)` can be replaced by `index`.

Reply:
You're right, my bad. But it has to be `index + 1` unless we change scheme to "Item 0", "Item 1" etc.
Will update.